widgetastic_manageiq/__init__.py

```diff
@@ -1089,6 +1091,119 @@ def workaround_save_issue(self):
             "{}._handlers.change.map(function(handler) {{ handler() }});".format(self.item_name))
 
 
+class SSUIVerticalNavigation(VerticalNavigation):
+    """The Patternfly Vertical navigation."""
+    CURRENTLY_SELECTED = './/li[contains(@class, "active")]/a/span'
+
+    @property
+    def currently_selected(self):
+        for el in self.browser.elements(self.CURRENTLY_SELECTED, parent=self):
+            return self.browser.text(el)
+
+
+class SSUIlist(Widget, ClickableMixin):
+    """Represents the list of items like Services ,
+    Orders or service catalogs in SSUI pages.
+    """
+
+    LOCATOR = '//div[@id={}]/../div//a[normalize-space(.)={}]'
+
+    def __init__(self, parent, list_name, logger=None):
+        Widget.__init__(self, parent, logger=logger)
+        self.list_name = list_name
+
+    def locator(self, item_name):
+        return self.LOCATOR.format(quote(self.list_name), quote(item_name))
+
+    def click_at(self, item_name):
+        """Clicks the list item with this name.
+
+        Args:
+            item_name: Name of the item
+        """
+        return self.browser.click(self.locator(item_name))
+
+
+class SSUIDropdown(Dropdown):
+    """Represents the SSUI dropdown."""
+
+    ROOT = ParametrizedLocator(
+        '//button[contains(@class, "dropdown-toggle")]/span[normalize-space(.)="Configuration"]')
```

Comment:
REQUIRED: Always start your xpath locators with `.` to ensure they are not looked up on whole page but just only in the context of the container. That applies for all of them.

Reply:
These locators do not work with . in front .

Reply:
Then are you looking up elements correctly in the DOM? That would mean you would be trying to access the elements in other branches of the DOM.